### PR TITLE
Add infobox for legacy channel fader option

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**
Adds a notice to all install pages about `--cleanuplegacyfadersettings`. I think that's the only missing thing for the docs. This message would just stay up until we release the next version. 

screenshot dark infobox

screenshot light infobox

**Context: Fixes an issue? Related issues**
Related to: https://github.com/jamulussoftware/jamulus/pull/2839

**Status of this Pull Request**
Waiting on review

**What is missing until this pull request can be merged?**
Nothing

**Does this need translation?**

YES

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

**Comment by ann0see** [comment]
I've rephrased the paragraph, so the screenshot is out of date (but the design should still be similar)

**Comment by pljones** [comment]
Can we have an "UpgradeNotice-Common.md" that gets included in all the files?  It can then have whatever is deemed appropriate in it (or be empty).  (The idea being it's one file to track for translation.)

**Comment by ann0see** [comment]
Yes, I hope translation would pick it out automatically

**Comment by ann0see** [comment]
I've made it a bit more generic now. It's easier to be a .html instead of .md

File: wiki/en/misc/general.yml [hunk 1]
```diff
@@ -22,3 +22,4 @@ kb:
   discuss:
     title:           "Comments"
     linktitle:       "Discuss this content"
+infobox_each_os:     'Jamulus 3.9.1 addresses a MIDI controller-related bug that could result in the loss of old fader levels: If you used `--ctrlmidich`, you can start the app once with `--cleanuplegacyfadersettings` to try to convert the old format to the new one. This option will be removed in a future release. Read the <a href="https://github.com/jamulussoftware/jamulus/blob/master/ChangeLog">Change Log</a> for more information.'
```
Comment (pljones) [inline wiki/en/misc/general.yml:25]: Will this get picked up for translation?  Isn't it simpler just to put the content into `_includes/infobox_each_os.html`?

Comment (ann0see) [reply wiki/en/misc/general.yml:25]: It will. And it's much easier than writing .po(t) files for _includes, I think.

**Comment by ann0see** [comment]
@pljones is this ready to merge?

**Comment by pljones** [comment]
Ah, it's still "In progress" rather than "Waiting on Team", that's why I missed it.  Yes, I should have checked.

File: wiki/en/misc/general.yml (hunk 1 above)
Comment (pljones) [reply wiki/en/misc/general.yml:25]: Maybe have it as a yaml multiline string, then.  Like this looks quite awkward.

But I've no strong feeling on it.

**Review by pljones — APPROVED** [review]

**Comment by ann0see** [comment]
Sorry about forgetting to move it.

File: wiki/en/misc/general.yml (hunk 1 above)
Comment (ann0see) [reply wiki/en/misc/general.yml:25]: As almost nobody looks at it here anyway, I think it's easier to have a long string in here.

**Pull request merged**